fetchUserPreferencesSchema.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-gq https://user-prefs-development.herokuapp.com/v1/graphql --introspect >common-queries/src/clue/resources/UserPreferencesDB.graphql
```

Comment:
Maybe we want to keep referencing `development` here?

Reply:
Not sure, do we normally get the schema from dev, I'd think we only update it when we change it on master?

Reply:
I was under the impression that development is for unmerged changes and master is for merged ones.

In other words, when we make a change, we change it on development first, and fetch it so as to be able to work on explore.

When the PR is merged we promote the development hasura to master. If we promote earlier we might break it for other developers using master.

That said, I see that the development environment in `environments.conf.json` points to development instead of master. Should we change this? Otherwise, the names are a bit confusing.

Reply:
I thought it was the other way around 😄

Reply:
I guess we can revert in another PR if needed